packages/middleware-sdk-sts/package.json

```
@@ -0,0 +1,49 @@
+{
+  "name": "@aws-sdk/middleware-sdk-sts",
+  "version": "3.0.0",
+  "scripts": {
+    "prepublishOnly": "yarn build:cjs && yarn build:es",
+    "build:cjs": "tsc -p tsconfig.cjs.json",
+    "build:es": "tsc -p tsconfig.es.json",
+    "build": "yarn build:es && yarn build:cjs",
+    "postbuild": "downlevel-dts dist/types dist/types/ts3.4",
```

downlevel-dts should move to prepubishOnly https://github.com/aws/aws-sdk-js-v3/pull/2218
```suggestion
    "prepublishOnly": "yarn build && downlevel-dts dist/types dist/types/ts3.4",
    "build:cjs": "tsc -p tsconfig.cjs.json",
    "build:es": "tsc -p tsconfig.es.json",
    "build": "yarn build:es && yarn build:cjs",
```